pkg/jitter/buffer.go

```diff
@@ -61,24 +63,45 @@ func (b *Buffer) Push(pkt *rtp.Packet) {
 	b.mu.Lock()
 	defer b.mu.Unlock()
 
-	var start, end bool
+	var start, end, padding bool
 	if len(pkt.Payload) == 0 {
 		start = true
 		end = true
+		padding = true
+		// drop padding packets from the beginning of the stream
+		if !b.initialized {
+			return
+		}
 	} else {
 		start = b.depacketizer.IsPartitionHead(pkt.Payload)
 		end = b.depacketizer.IsPartitionTail(pkt.Marker, pkt.Payload)
 	}
 
-	p := b.newPacket(start, end, pkt)
+	p := b.newPacket(start, end, padding, pkt)
+
+	beforePrev := before(pkt.SequenceNumber, b.prevSN)
+	outsidePrevRange := outsideRange(pkt.SequenceNumber, b.prevSN)
+
+	// drop if packet comes before previously pushed packet
+	if b.initialized && beforePrev && !outsidePrevRange {
+		b.logger.Debugw("packet dropped",
```

Comment:
if it's a padding packet, I think we shouldn't trigger dropped.. until the next non-padding packet.